atomic_reactor/koji_util.py

```diff
@@ -96,3 +98,22 @@ def wait(self):
 
     def failed(self):
         return self.state in ['CANCELED', 'FAILED']
+
+
+def stream_task_output(session, task_id, file_name,
+                       blocksize=DEFAULT_DOWNLOAD_BLOCK_SIZE):
+    """
+    Generator to download file from task without loading the whole
+    file into memory.
+    """
+    logger.debug('Streaming {} from task {}'.format(file_name, task_id))
```

Comment:
Not worth fixing unless something else comes up, but landscape.io suggests using '%s' here and passing file_name and task_id as parameters, e.g. logger.debug("Streaming %s from task %s", file_name, task_id) -- just means it doesn't have to format the string if debugging isn't turned on.

Same a few lines down too.